helm/vmss-prototype/templates/vmss-prototype.yaml

```diff
@@ -10,6 +10,35 @@
 {{- end -}}
 {{- end -}}
 
+apiVersion: v1
```

Comment:
Normally one would put these in their own files rather than making a multi-part yaml but this will work if needed.